README.md

```diff
@@ -30,25 +20,40 @@ Or install it yourself as:
 
     $ gem install sequel-bulk-audit
 
-After Installation you should run ```rails g audit_migration``` generator.
+After installation you should run ```rails g audit_migration``` generator.
 
 You can exdend this migration by attaching the trigger to audited tables.
 
+Please note, that this gem reqires pg_array and pg_json sequel extensions to work.
+
 ## Usage
 
-Models, changes in which you plan to audit should contain
+Models with audited changes should contain:
+
 ```ruby
 plugin :bulk_audit
 ```
 
-Method #with_current_user should wrap all the operations on the table.
+Method #with_current_user should wrap all the operations on the table. You must use method from the model you are changing for this gem to work correclty. 
+
+Keep in mind that everything wraped in #with_current_user will happen in one transaction.
+
+Correct usage:
 
 ```ruby
 Model.with_current_user(current_user) do
   Model.where(...).update(...)
 end
 ```
 
+Incorrect usage:
+
+```ruby
+SomeOtherModel.with_current_user(current_user) do
+  Model.where(...).update(...)
```

Comment:
do not we want to describe with more details the situation with audit several models in the same transaction?

Reply:
and it will be great to have the test case for this situation.

Reply:
Done.